examples/stable-diffusion/quantization/measure_config.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
if we are adding quantization folder to SD examples, we should put it in subdir for sd3, since we then can add the config files for other SD pipelines

Reply:
done